pages/member.tsx

```diff
@@ -0,0 +1,35 @@
+import { Loading } from 'idea-react';
+import { observer } from 'mobx-react';
+import { InferGetServerSidePropsType } from 'next';
+import { FC } from 'react';
+import { Container } from 'react-bootstrap';
+
+import { MemberList } from '../components/Member/List';
+import { PageHead } from '../components/PageHead';
+import memberStore, { MemberModel } from '../models/Member';
+import { i18n } from '../models/Translation';
+import { withTranslation } from './api/core';
+
+export const getServerSideProps = withTranslation(async () => {
+  const list = await new MemberModel().getList({}, 1);
+
+  return { props: { list } };
+});
```

Comment:
```suggestion
export const getServerSideProps = withErrorLog(withTranslation(async () => {
  const list = await new MemberModel().getList();

  return { props: { list } };
}));
```
1. 有接口请求的最好都加上错误日志中间件，便于页面报 500 时做调试
2. 新实例默认加载的就是第一页